devlib/trace/perf.py

```diff
@@ -11,131 +11,128 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
 
+# pylint: disable=missing-docstring
 
+import collections
 import os
-import re
-from past.builtins import basestring, zip
+import sys
 
+from devlib.utils.cli import Command
 from devlib.host import PACKAGE_BIN_DIRECTORY
 from devlib.trace import TraceCollector
-from devlib.utils.misc import ensure_file_directory_exists as _f
 
+if sys.version_info >= (3, 0):
+    from shlex import quote
+else:
+    from pipes import quote
```

Comment:
This implies that compatibility with Python2.7 is attempted to be retained however is not present elsewhere in this PR. Is this aimed to be merged before the next release aka supporting Python 2.7 or to be held until after support has been dropped?